tests/integration/security/tls_origination/main_test.go

```diff
@@ -0,0 +1,59 @@
+//  Copyright Istio Authors
```

Comment:
Could you move tls_origination/ into security/egress/filemount_tls_origination ?

Reply:
I don't think I can create another subsuite like that. security/filebased_tls_origination can be created but filebased_tls_origination suite in security/egress cannot